lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -469,13 +469,19 @@ def make_latest_pki_issuer_default(self, mount: str) -> None:
         except (InvalidPath, KeyError) as e:
             logger.error("No issuers found on the specified path: %s", e)
             raise VaultClientError("No issuers found on the specified path.")
-        self._client.write_data(
-            path=f"{mount}/config/issuers",
-            data={
-                "default_follows_latest_issuer": True,
-                "default": first_issuer,
-            },
-        )
+        try:
+            issuers_config = self._client.read(path=f"{mount}/config/issuers")
+            if issuers_config and not issuers_config["data"]["default_follows_latest_issuer"]:  # type: ignore -- bad type hint in stubs
```

Comment:
I'm assuming by the way this is written, `issuers_config` exists as some default value even before we write anything?

Reply:
Yes, my intention is to wait until Vault creates the config then we modify it.